### PR TITLE
Move disable chronyd to tuned patch

### PR DESCRIPTION
**Pull request by tliu2021**
This update removes the machine config that disables chronyd and adds disabling
chronyd service in the tuned profile.

Signed-off-by: tliu2021 <tali@redhat.com>

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/Upgrade.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ This section only applies when upgrading from an earlier release to 4.10. In rel
 
 All PolicyGenTemplates must be created in a Namespace prefixed with "ztp". This ensures that the GitOps ZTP application is able to manage the Policy CRs generated by GitOps ZTP without conflicting with the way ACM manages the policies internally.
 
+##### Add disabling chronyd to PolicyGenTemplate CR
+
+Add disabling chronyd in “service” section to the overlaid spec file of TunedPerformancePatch.yaml as shown below. This is normally included in the group policies. [service] service.stalld=start,enable service.chronyd=stop,disable
```
Comment (Missxiaoguo) [inline ztp/gitops-subscriptions/argocd/Upgrade.md:74]: Probably start a new line for the "service" section so it aligns with the 4.11 doc?

Comment (tliu2021) [reply ztp/gitops-subscriptions/argocd/Upgrade.md]: Done.

**Pull request merged**